app/component/ItineraryTab.js

```diff
@@ -354,6 +385,52 @@ class ItineraryTab extends React.Component {
                 />
                 {config.showRouteInformation && <RouteInformation />}
               </div>
+              {config.showCO2InItinerarySummary && co2value >= 0 ? (
+                <div className="itinerary-co2-comparison">
+                  <div className="itinerary-co2-line">
+                    <div className={cx('divider-top')} />
+                    <div className="co2-container">
+                      <div className="co2-description-container">
+                        <Icon img="icon-icon_co2_leaf" className="co2-leaf" />
+                        <span className={cx("itinerary-co2-description", { simple: co2SimpleDesc })}>
+                          {co2SimpleDesc ?
+                            <FormattedMessage id="itinerary-co2.description-simple"
+                              defaultMessage="CO₂ emissions for this route2"
+                              values={{
+                                co2value,
+                              }}
+                            />
+                            :
+                            <FormattedMessage
+                              id="itinerary-co2.description"
+                              defaultMessage="CO₂ emissions for this route"
+                              values={{
+                                co2value,
+                                carCo2Value,
+                              }}
+                            />
+                          }
+                          {config.URL.EMISSIONSINFO && (
+                            <div className='co2link'>
+                              <Link style={{ textDecoration: 'none', fontWeight: '450' }}
```

Comment:
Link from found library should only be used for links inside this app. Just use `<a>` tag here. The link doesn't work currently, I'm not sure if it is just because of this issue.

Reply:
Link wasn't working because of the lang variable. Changed Link to <a> tag also.